tests/baselines/reference/aliasOnMergedModuleInterface.errors.txt

```diff
@@ -8,7 +8,7 @@ tests/cases/compiler/aliasOnMergedModuleInterface_1.ts(5,16): error TS2304: Cann
     z.bar("hello"); // This should be ok
     var x: foo.A = foo.bar("hello"); // foo.A should be ok but foo.bar should be error
                    ~~~
-!!! error TS2304: Cannot find name 'foo'.
+!!! error TS2691: Cannot find name 'foo'. A type exists with this name, but no value.

```

Comment:
Somehow this seems like a regression because `foo` is also a namespace (i.e. a container of types and other namespaces).

What you potentially want to do is make sure that the resolved entity is not a namespace when giving this error. Aliases (imports) are okay though since you really care more about what's being aliased.